m4_sessions/1808-inning/surprise_of_happiness.md

```diff
@@ -1,18 +1,30 @@
 ## Group Members:
+Haley Mesander
+Sabrina Robinson
+Tory Dannenberg
 
-
-
-## Name of Session: 
+## Name of Session: Surprise of Happiness
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+     What makes you happy? That's what you think...
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+     More useful ways to benefit your pom (fun exercise challenges, going outside, listening to a song). 
+
+
 
 ## Session Outline 
+     10 min for intros
+     5 min for pairing
+     5 min quiz
+     5 min to discuss quiz that they relate with
+     5 min to discuss quiz that they didn't relate with
+     10 min for video
+     10 min pom
+     5 min discussion about video
```

Comment:
Is the discussion about the video also the closing?  Like a closing discussion?  I want you to push students to synthesize/summarize what they are taking away or what they will do differently because of this session.  What will they commit to?